### PR TITLE
Basic loan table view

### LINKED ISSUE #5: Basic table view
We need to create a very basic table view to display loan information. For this issue, the goal is just to display loans and we'll look to add searching, filtering, etc. in other tickets.

Acceptance Criteria:
- Basic table view created
  - Part of this will be deciding what we actually want to display in the table since there's too many fields in our db to display everything
  - Should call endpoint built in #3

### PR DESCRIPTION
**Pull request by skyripley**
## Background
- This PR addresses issue #5 

## What is changing?
- Adds really basic table view
- Adds top navigation bar
- Adds routing so that we can route to different views (ex `/table` now routes to the `TableView` component), and sets up routes for each of our expected views (i.e. table, statistics, and search)

## How is this tested?
- Tested locally

## Notes
- I removed the install of react-scripts in the frontend dockerfile because it was taking a really long time and I noticed that it's completely unnecessary if we use the most up to date `alpine-node`
- The table here doesn't include any pagination, but I'll open a ticket to add that. I decided not to add it here because our initial demo is coming up in a couple of weeks and I wanted to be 100% certain that we have at least one component to demo.
- I have no react experience so let me know if things can be done in better ways!

### CODE REVIEW

File: backend/config/settings.py [hunk 1]
```diff
@@ -27,6 +27,8 @@
 
 ALLOWED_HOSTS = ['*']
 
+CORS_ORIGIN_ALLOW_ALL = True
```
Comment (skyripley) [inline backend/config/settings.py]: Had to add this to be able to call our api from the frontend container. Anyone know of a better restriction though?

File: frontend/src/api/LoanApi.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export default {
+    getLoans() {
+        return fetch('http://localhost:8000/api/loan/')
```
Comment (skyripley) [inline frontend/src/api/LoanApi.js:3]: I don't like hard-coding the api url, but didn't seem like there was an easy method that I could use for a quick win here. Anyone know of a more dynamic way to get this?

File: backend/config/settings.py (hunk 1 above)
Comment (Skyblueballykid) [reply backend/config/settings.py]: Do you think it matters to restrict it? I guess it is best practice. A quick google looks like we could pull in another dependency... I found this stackoverflow post which talks about the django_cors_headers package:

https://stackoverflow.com/questions/22355540/access-control-allow-origin-in-django-app-when-accessed-with-phonegap

https://github.com/adamchainz/django-cors-headers

**Review by Skyblueballykid — COMMENTED** [review]
Skylar, your changes build for me and render. The /table endpoint only renders for a second though and then it immediately redirects to a blank screen. I haven't figured out what the issue is yet, will update this later if I figure it out. /statistics and /search do render the Navbar correctly. Is it because there is no data loaded yet and so it doesn't have anything to render? Still weird that it doesn't render the Navbar...

File: backend/config/settings.py (hunk 1 above)
Comment (skyripley) [reply backend/config/settings.py]: Might not need to restrict it since this is just a grad school project. We're actually already using that dependency.

Comment (Skyblueballykid) [reply backend/config/settings.py]: Yea, whatever you prefer. We can do it and if it becomes a pain we can open it back up.

**Comment by skyripley** [comment]
> Skylar, your changes build for me and render. The /table endpoint only renders for a second though and then it immediately redirects to a blank screen. I haven't figured out what the issue is yet, will update this later if I figure it out. /statistics and /search do render the Navbar correctly. Is it because there is no data loaded yet and so it doesn't have anything to render? Still weird that it doesn't render the Navbar...

@Skyblueballykid yeah if you don't have any data then I think it's probably causing an error on the frontend side because we weren't handling that case. That's a good catch though because I think it should still display the table, but with no rows so I added 68c38f6 to fix it.

File: backend/config/settings.py (hunk 1 above)
Comment (skyripley) [reply backend/config/settings.py]: Sounds good, I added c4d3703 to restrict allowed origins to localhost. That's a good point, if it's a hassle we can allow all again in the future.

**Pull request merged**